lib/chef_zero/rspec.rb

```diff
@@ -227,9 +227,16 @@ def cookbook(name, version, data = {}, options = {}, &block)
         end
       end
 
-      def cookbook_artifact(name, identifier, data, &block)
+      def cookbook_artifact(name, identifier, data = {}, &block)
         with_object_path("cookbook_artifacts/#{name}") do
-          ChefZero::RSpec.server.load_data({ 'cookbook_artifact' => { name => { identifier => data } } }, current_org)
+          if data.has_key?('metadata.rb')
+            if data['metadata.rb'].nil?
+              data.delete('metadata.rb')
```

Comment:
Why do we ensure that there is at least a minimal metadata `data['metadata.rb'] = "name #{name.inspect}"` if there is no `metadata.rb` key, but coerce the data to not have a `metadata.rb` key if it's null?